backend/README.md

```diff
@@ -53,3 +52,36 @@ From the `backend` directory:
 ```
 yarn deploy
 ```
+
+# Python cloud functions
+
+[Python cloud functions](https://cloud.google.com/functions/docs/quickstart-python) are not
+administered by Firebase, though they are associated
+with the same project specified above, so you will need the same access to administer them.
+
+They live in `backend/python-functions/main.py`.
+
+## Testing locally
+
+In `python-functions` set up your virtual environment:
+
+```sh
+virtualenv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+Unlike Firebase, we don't have tools to emulate these functions locally.
```

Comment:
But to be clear, when you run the app locally it will still call out to the deployed Python function, so if you're working on anything other than the function itself, there's no concern, right?

Reply:
that's right. i am working on a subsequent PR to set that up